spec/factories/file_attachment_metadata_revision_factory.rb

```diff
@@ -2,5 +2,6 @@
   factory :file_attachment_metadata_revision, class: "FileAttachment::MetadataRevision" do
     association :created_by, factory: :user
     title { SecureRandom.hex(8) }
+    unique_reference { SecureRandom.alphanumeric(10) }
```

Comment:
Minor: it's a bit strange having a non-required attribute in the factory, and could lead to subtle bugs where it's presence is assumed somewhere.

Reply:
Updated.